include/sisl/cache/evictor.hpp

```diff
@@ -64,13 +70,14 @@ class Evictor {
 
     int64_t max_size() const { return m_max_size; }
     uint32_t num_partitions() const { return m_num_partitions; }
-    const can_evict_cb_t& can_evict_cb(const uint32_t record_id) const { return m_can_evict_cbs[record_id].second; }
+    const eviction_cb_t& can_evict_cb(const uint32_t record_id) const { return m_eviction_cbs[record_id].second.can_evict_cb; }
```

Comment:
Can you put brief description for these two APIs, especially why we need the post_evicttion_cb?